src/components/FieldValueDirectAnswer.tsx

```diff
@@ -105,9 +105,13 @@ function getResultContent(
     case BuiltInFieldType.Address:
       return getAddressJsxElement(result.value);
     case BuiltInFieldType.RichText:
-      return <div>{result.value}</div>; //TODO: SLAP-2340
+      //TODO: SLAP-2340
+      console.warn('Rendering markdown for rich text direct answer is currently not supported. Displaying the unrendered markdown string as is.');
```

Comment:
nit: since this could be string array, maybe this warning should be "unrendered markdown string(s) as is"